lib/watir/rspec/cli.rb

```diff
@@ -88,6 +88,8 @@ def install
 end
   ]
           end
+
+          puts "WARNING : the HTML formatter cannot be used until the rspec legacy formatter gem is also installed !! (https://github.com/rspec/rspec-legacy_formatters)"
```

Comment:
I converted the HTML formatter to rspec version 3 , so the legacy gem is no longer needed